src/event/views/EventPage.vue

```diff
@@ -32,13 +32,15 @@
           />
           <div class="buttons">
             <AppButton
+              :class="'btn-secondary'"
               class="btn"
-              text="Save Response"
+              content="Save Response"
```

Comment:
Since we're not using the content prop anymore, you can just pass in the "Save Response" as text, like so:

```
<AppButton>Save Response</AppButton>
```

This also lets us nest the icon in the button like the example I gave in #65 :)